src/commands/_common.ts

```diff
@@ -71,7 +72,7 @@ export async function getSettings(options: Options = {}): Promise<Settings> {
     const relativePath = resolve(process.cwd(), path);
 
     try {
-      return require(relativePath);
+      return relativePath.endsWith(".mjs") ? import(relativePath) : require(relativePath);
```

Comment:
For the import error to be caught you must await the import:

```suggestion
      return relativePath.endsWith(".mjs") ? await import(relativePath) : require(relativePath);
```

Reply:
Ouh, well spotted indeed.